core/src/js/components/main-nav/NavItem.js

```diff
@@ -140,10 +145,13 @@ export default class NavItem {
    */
   closeSubNav(focusOnTrigger = false) {
     if (this.isSubNavTrigger()) {
-      this.item.classList.remove('su-main-nav__item--expanded');
-      this.setExpanded('false');
-      if (focusOnTrigger) {
-        this.link.focus();
+      if (this.isExpanded()) {
```

Comment:
Why do we need to check if the item is expanded here? Is it destructive to execute the close items if it wasn't open?

Reply:
See comment above about `closeMobileNav()`. Same reasoning applies here.

Reply:
Thanks, GTG here.